pkg/server/k8s/client.go

```diff
@@ -492,6 +494,59 @@ func (k *k8sClient) CreateService(namespace, appName string) error {
 	return errors.Wrap(err, "create service failed")
 }
 
+func (k *k8sClient) hasIngress(namespace, appName string) (bool, error) {
+	kc, err := k.buildClient()
+	if err != nil {
+		return false, err
+	}
+	_, err = kc.ExtensionsV1beta1().Ingresses(namespace).Get(appName)
+	if err != nil {
+		if k.IsNotFound(err) {
+			return false, nil
+		}
+		return false, errors.Wrap(err, "get ingress failed")
+	}
+	return true, nil
+}
+
+func (k *k8sClient) createIngress(namespace, appName, host string) error {
+	kc, err := k.buildClient()
+	if err != nil {
+		return err
+	}
+	igsSpec := ingressSpec(namespace, appName, host)
+	_, err = kc.ExtensionsV1beta1().Ingresses(namespace).Create(igsSpec)
+	return errors.Wrap(err, "create ingress failed")
+}
+
+// ExposeApp create a service and/or a ingress if needed
+func (k *k8sClient) ExposeApp(namespace, appName, host string, w io.Writer) error {
+	hasSrv, err := k.hasService(namespace, appName)
+	if err != nil {
+		return err
+	}
+	if !hasSrv {
+		fmt.Fprintln(w, "Exposing service")
+		if err := k.createService(namespace, appName); err != nil {
+			return err
+		}
+	}
+
+	if k.ingress {
```

Comment:
```diff
- if k.ingress {
+ if !k.ingress{
+     return nil
+ }
```

Reply:
I'm not sure about that, but I no like this approach

Reply:
why not? with this minor change you can decrease one level of indentation

Reply:
looks unnecessary, I think that this way is more cohesive

Reply:
I disagree, but I like to listen to @aguerra opinion.

Reply:
It's standard idiom in Go to treat the default case as not indented.

Reply:
updated.